.env.example

```
@@ -1 +1,2 @@
 SECURITY_HASH_PEPPER=
+SECURITY_ENC_KEY_BASE64=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)